app/design/adminhtml/default/default/template/catalog/product/edit/options/option.phtml

```diff
@@ -73,7 +73,7 @@ var productOption = {
         }
 
         //set selected is_require
-        if (data.is_require) {
+        if (typeof data.is_require !== 'undefined') {
```

Comment:
How about this:
```suggestion
        if (data.hasOwnProperty('is_require')) {
```
Because we want to know if the property exists, if it does, the check `if (option.value==data.is_require) option.selected = true;` will work even if the value of `data.is_require` is `undefined`.

Reply:
ok done